pkg/operator/ceph/cluster/osd/config/scheme.go

```diff
@@ -379,7 +379,8 @@ func (e *PerfSchemeEntry) GetPartitionArgs() []string {
 	args := []string{}
 	partNum := 1
 
-	if collocated && e.StoreType == Bluestore {
+	// A device will use bluestore unless explicitly requested to be filestore (the default is blank)
+	if collocated && e.StoreType != Filestore {
```

Comment:
is there a unit test we can add/update for this fix?

i'm wondering if there's still some lurking issues regarding having a blank store type by default.  is there anywhere else we need to worry about this in the codebase?

Reply:
I did check the tests for scheme and found we are testing this part of the code. Will give it another check to see if i missed anything. For the rest of the code-base, I can give it a check and see if there are other places. Though my proficiency of the code is not the best.

Reply:
I appreciate that @ganeshmaharaj, let us know if you have any questions about what you find.  and let me know when there's a test addition to take a look at. thanks!